static/scss/pages/dashboard.scss

```diff
@@ -304,20 +385,310 @@
     margin-left: auto;
 }
 
-.c-alias-additional-info {
-    border-top: 1px solid transparent;
+// The email delivery label is hidden on mobile
+.c-alias-block-levels {
+    display: none;
+    align-self: center;
+
+    @media #{$mq-md} {
+        display: block;
+    }
+}
+
+.c-alias-block-level {
+    display: none;
+    border: 2px solid;
+    border-radius: $border-radius-sm;
+    padding: $spacing-xs $spacing-md;
+    font-weight: 600;
+
+    @include text-body-sm;
+
+    &.t-forward {
+        color: $color-green-50;
+        border-color: $color-green-30;
+    }
+
+    &.t-critical {
+        color: $color-yellow-50;
+        border-color: $color-yellow-40;
+    }
+
+    &.t-block {
+        color: $color-grey-30;
+        border-color: $color-grey-10;
+    }
+}
+
+.c-alias-info {
     overflow: hidden;
     max-height: 0;
     transition: all .2s ease-out;
+    padding: 0;
+
+    .c-alias-block-settings, .c-alias-additional-info {
+        border-top: 1px solid transparent;
+    }
     
     @media #{$mq-sm} {
-        display: flex;
+        display: block;
     }
 
     &.is-visible {
-        border-color: $color-light-gray-30;
+
+        .c-alias-block-settings, .c-alias-additional-info {
+            border-color: $color-light-gray-30;
+        }
         max-height: 900px;
-        padding: $spacing-md 0;
+    }
+}
+
+.c-alias-additional-info {
+    display: flex;
+    padding: $spacing-md;
+}
+
+.c-alias-block-settings {
+    border-top: 1px solid transparent;
+    border-color: $color-light-gray-30;
+    padding: $spacing-md;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &.is-hidden {
+        display: none;
+    }
+
+    @media #{$mq-md} {
+        display: flex;
+        justify-content: space-between;
+        flex-direction: row;
+        min-height: 120px;
```

Comment:
This can be removed. It was in place before the copy "What emails do you want to block? " was added above the block setting buttons on desktop to keep the content vertically centered. 

Removed in  e98ce2ad296909333271ce2460aebb63c189b09f